docs/sources/operations/storage/retention.md

```diff
@@ -6,13 +6,14 @@ weight:  600
 ---
 # Log retention
 
-Retention in Grafana Loki is achieved either through the [Table Manager](#table-manager) or the [Compactor](#compactor).
+Retention in Grafana Loki is achieved through the [Compactor](#compactor).
+It is the recommended way of applying retention for [boltdb-shipper]({{< relref "./boltdb-shipper" >}}) and [tsdb]({{< relref "./tsdb" >}}).
```

Comment:
Should we call it "single store BoltDB" and "single store TSDB", like we do on their respective pages?